lib/embulk/input/google_analytics/plugin.rb

```diff
@@ -102,8 +102,8 @@ def run
 
           latest_time_series = nil
           client.each_report_row do |row|
-            time = row[task["time_series"]]
-            next if last_record_time && time <= last_record_time
+            time = row["ga:dateHour"] ? row["ga:dateHour"] : row["ga:date"]
+            next if last_record_time && time.utc <= last_record_time.utc
```

Comment:
The purpose of this PR is checking data from `ga:dateHour` or `ga:date` with `last_record_time` instead of using `time_series` for checking.

About `.utc`, i only make sure comparing time with same timezone is UTC. But maybe its redundant, i will remove it